src/lib/components/workflow/workflow-typed-error.svelte

```diff
@@ -163,35 +172,53 @@
     ) {
       return 'WorkflowTaskHeartbeatError';
     }
+
     return cause;
   }
 
   export let error: WorkflowTaskFailedEvent;
 
-  $: cause = getErrorCause(error);
-  $: errorCopy = WORKFLOW_TASK_FAILED_ERROR_COPY[cause] ?? {};
-  $: ({ title = '', copy = '', actionCopy = '', link = '' } = errorCopy);
+  let title: I18nKey<'typed-errors'>;
+  let description: I18nKey<'typed-errors'>;
+  let action: I18nKey<'typed-errors'> | undefined;
+  let link: I18nKey<'typed-errors'> | undefined;
+
+  $: {
+    if (error) {
+      const cause = getErrorCause(error);
+      const errorCopyKeys = WORKFLOW_TASK_FAILED_ERROR_COPY_KEYS[cause];
```

Comment:
This is true the way it was written previously, but by wrapping these calls in `if (error)`, `cause` and `errorCopyKeys` will always be defined (thus no TS error).

Reply:
I see. Well by wrapping the `getErrorCause` call in `if (error)`, the `if (!error ...)` in that function is redundant. Is there a default cause we can use instead of returning `undefined` from `getErrorCause` ?

Reply:
I'd hesitate to return a default cause since that could be misleading. Probably better to not show the typed error at all 🤔